source/_integrations/samsungtv.markdown

```diff
@@ -94,6 +96,53 @@ media_content_type: channel
 It's possible to switch between the 2 sources `TV` and `HDMI`.
 Some older models also expose the installed applications through the WebSocket, in which case the source list is adjusted accordingly.
 
+### Remote
+
+The integration supports `remote` platform. The remote allows you to send key commands to your TV with the `remote.send_command` service.
+
+The exact supported keys varies between TV models. See [this link](https://github.com/jaruba/ha-samsungtv-tizen/blob/master/Key_codes.md) for possible keys.
```

Comment:
```suggestion
The exact supported keys vary between TV models. See [this link](https://github.com/jaruba/ha-samsungtv-tizen/blob/master/Key_codes.md) for possible keys.
```

Reply:
I'm not sure why we have to link into an external GitHub page?
We already document it below, right?

Reply:
This isn't the full key set by any stretch. The full list is more than 200 keys 🙂

Reply:
Isn't that an issue already? You now link to an external page, which is in GitHub, that imposes the same full list? How is that better?

Reply:
Sorry, I'm not following the issue here.
I've not duplicated the full list, I've merely included a very small sample.

Are you suggesting we shouldn't link to the complete list of key codes here? Or are you suggesting we copy all 200+ inline here? 😅

Reply:
IMHO both are bad? 

We can just add all of them and put them in a details block so their are only shown when a user expands it.

../Frenck

Reply:
Done!